heat/core/operations.py

```diff
@@ -383,7 +378,7 @@ def __local_operation(operation, x, out):
     operation(casted.repeat(multiples) if needs_repetition else casted, out=out._tensor__array)
     return out
 
-def __reduce_op(x,partial, op, axis):
+def __reduce_op(x,partial_op, op, axis):
```

Comment:
Please add a space before partial_op to be PEP-8 compliant

Reply:
Hi Claudia, there are automatic tools taking care of PEP-8 compliance, e.g. autopep8, also some IDEs
like pycharm have these feature.